PalasoUIWindowsForms.Tests/SettingsProviderTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using NUnit.Framework;
+using Palaso.IO;
+using Palaso.Settings;
+using Palaso.TestUtilities;
+using Palaso.UI.WindowsForms.Registration;
+
+namespace PalasoUIWindowsForms.Tests
+{
+	[TestFixture]
+	public class SettingsProviderTests
+	{
+		[Test]
+		public void SettingsFolderWithNoConfig_SortsAfterOneWithConfig()
+		{
+			// We do two iterations like this because we can't count on deleting the file to really make it gone at once.
+			for (var which = 0; which < 1; which++)
```

Comment:
This comment made me expect to see a file deletion. I understand the problem you are avoiding but maybe you can re-word this to indicate that we are NOT deleting files, but switching which folder does not have a config.